src/views/compute/wizard/NetworkConfiguration.vue

```diff
@@ -25,17 +25,45 @@
     size="middle"
     :scroll="{ y: 225 }"
   >
+    <template slot="name" slot-scope="text, record">
+      <div>{{ text }}</div>
+      <small>{{ $t('label.cidr') + ': ' + record.cidr }}</small>
+    </template>
     <template slot="ipAddress" slot-scope="text, record">
-      <a-input
-        style="width: 150px;"
-        :placeholder="$t('label.ipaddress')"
-        @change="($event) => updateNetworkData('ipAddress', record.id, $event.target.value)" />
+      <a-form-item>
```

Comment:
@utchoang Could you hide this if the network typle is L2, since they do not have IP addresses ?